docs/pages/database-access/guides/aws-discovery.mdx

```diff
@@ -157,6 +157,22 @@ automatically.
 You can list them with the `tctl get db` command or check a specific database
 with `tctl get db/<database-name>`.
 
+<Admonition type="note" title="database names">
```

Comment:
Since this Admonition is almost as long as the main body content in Step 4, I would restructure this H2 to make this content more discoverable:
- Change "Start the Discovery Service" to something like "Use the Discovery Service" or "Cccess databases".
- Add an H3 for starting the discovery service with everything until `tctl get db`.
- Add an H3 called "List registered databases" or something like that, which includes the `tctl get db` command as well as the content in the Admonition.

Reply:
should I drop the `Admonition` after moving it in h3?